src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -119,15 +183,30 @@ private void getMealById(HttpServletRequest request, HttpServletResponse respons
         return;
     }
 
-     private void returnIdOfSimilar(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    private ArrayList<Meal> getDataFromDatastore(PreparedQuery query) {
+        ArrayList<Meal> result = new ArrayList<>();
+        for (Entity entity : query.asIterable()) {
+            Long id = (Long)entity.getProperty("id");
+            String title = (String) entity.getProperty("title");
+            String description = (String) entity.getProperty("description");
+            ArrayList<String> ingredients = (ArrayList<String>) entity.getProperty("ingredients");
+            String type = (String) entity.getProperty("type");
+
+            Meal meal = new Meal(id, title, description, ingredients, type);
+            result.add(meal);
+        }
+        return result;
+    }
+
+    private void returnIdOfSimilar(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // Fuction redirect to random page
         // TODO(grenlayk): implement suggestions algorithm here for Product Alpha
-        Random rand = new Random(); 
+        /*Random rand = new Random(); 
         int randomId = rand.nextInt(dishes.size()); 
         
         String gson = new Gson().toJson(randomId);
         response.setContentType("application/json");
-        response.getWriter().println(gson);
+        response.getWriter().println(gson);*/
```

Comment:
As Olga mentioned earlier - do not comment old code that is no longer needed, just delete it instead. We can retrieve the old code by browsing the commit history.